kits/agentic/founder-lens/README.md

```
@@ -0,0 +1,170 @@
+   ```bash
+   cp .env.example .env.local
+   ```
+   Then fill in `.env.local` with your real values (see table below).
+
+4. **Start the development server**
+   ```bash
+   npm run dev
+   ```
+
+5. **Open your browser**
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)